.github/workflows/test.yaml

```diff
@@ -796,7 +833,7 @@ jobs:
         key: "${{ github.job }}-results-${{ github.run_id }}-${{ matrix.group }}-${{ github.run_attempt }}"
 
   spread-nested:
-    needs: [unit-tests]
+    needs: [unit-tests, snap-builds]
```

Comment:
this job is changing in https://github.com/snapcore/snapd/pull/14032, we should skip any change in spread-nested